substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -134,14 +138,22 @@ def add_opens_to_all_unnamed(packageName):
     # Reflective access to java.nio.MappedByteBuffer.fd.
     'java.base/java.nio',
     # Reflective access to java.util.Bits.words.
-    'java.base/java.util']
+    'java.base/java.util',
+    'java.base/jdk.internal.logger',]
 GRAAL_COMPILER_FLAGS_MAP['11'].extend(add_opens_from_packages(java_base_opens_packages))
 
 # Reflective access to org.graalvm.nativeimage.impl.ImageSingletonsSupport.
 graal_sdk_opens_packages = [
-    'org.graalvm.sdk/org.graalvm.nativeimage.impl']
+    'org.graalvm.sdk/org.graalvm.nativeimage.impl',
+    'org.graalvm.sdk/org.graalvm.polyglot',]
 GRAAL_COMPILER_FLAGS_MAP['11'].extend(add_opens_from_packages(graal_sdk_opens_packages))
 
+graal_truffle_opens_packages = [
+    'org.graalvm.truffle/com.oracle.truffle.polyglot',
+    'org.graalvm.truffle/com.oracle.truffle.api.impl',]
+GRAAL_COMPILER_FLAGS_MAP['11'].extend(add_opens_from_packages(graal_truffle_opens_packages))
+ 
```

Comment:
One of the things Travis complains about is a trailing whitespace on this line (155)